src/sharding/Shard.js

```diff
@@ -112,16 +112,23 @@ class Shard extends EventEmitter {
     return this.process;
   }
 
+  /**
+   * Kill this shard.
```

Comment:
Shouldn't this say `Kills`?
The docstring below says so as well.

Although we are quite a bit inconsistent in that regard.